components/board/client.go

```diff
@@ -223,6 +223,10 @@ func (arc *analogClient) Read(ctx context.Context, extra map[string]interface{})
 	return int(resp.Value), nil
 }
 
+func (ac *analogClient) Write(ctx context.Context, value int, extra map[string]interface{}) error {
+	return errors.New("unimplemented")
```

Comment:
I think this needs to be implemented for modular boards to work correctly..is the implementation part of different ticket?

Reply:
Yes, assigned to alan.